utils/DegenEmbed.py

```diff
@@ -1,25 +1,36 @@
 import discord
+from datetime import datetime
 
 
-class DegenEmbed:
-    def __init__(self, title, description, logo="https://avatars.githubusercontent.com/u/1737241?v=4", color=0x1f6e9e):
-        self.embed_title = title
-        self.embed_description = description
-        self.embed_logo = logo
-        self.embed_color = color
-        self.embed = discord.Embed(title=self.embed_title, url="https://www.seattledegens.wtf", description=self.embed_description,
-                                   color=self.embed_color)
+class DegenEmbed(discord.Embed):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
 
-# Need to_dict for some embed internals.
-    def to_dict(self):
-        return self.embed.to_dict()
-
-    def create(self, thumbnail, title=None):
-        # I don't really like the author field atm.
-        ## self.embed.set_author(name=f"Get Fucked", icon_url=self.embed_logo)
-        self.embed.set_thumbnail(url=thumbnail)
-        self.embed_title = title
-        self.embed.set_footer(text="Made by xDDYx's darkest thoughts.")
+    def create(self, thumbnail=None, title=None):
+        if thumbnail:
+            self.set_thumbnail(url=thumbnail)
+        if title:
+            self.title = title
+        self.set_footer(text="Made by xDDYx's darkest thoughts.")
 
     def add_field(self, name, value, inline=False):
-        self.embed.add_field(name=name, value=value, inline=inline)
\ No newline at end of file
+        super().add_field(name=name, value=value, inline=inline)
+
+    def construct_full_embed(cls, games: object, title: object, color: object) -> object:
+        newembed = DegenEmbed(title=title, color=color)
+        try:
+            newembed.create(games[0].team['logo_url'], title)
+        except Exception:
+            newembed.create("https://krakenhockeyleague.com/hockey/images/teamlogos100/Degens.png", title)
+
+        for game in games:
+            if game.gametime <= datetime.now():
+                newembed.add_field(f'{game.away_team}: {game.away_score} '
+                                   f'{game.home_team}: {game.home_score}',
+                                   f'{game.gametime.strftime('%B %d')}',
```

Comment:
I don't see why this gametime should be any different than an upcoming game gametime

Reply:
I did this because it was too busy for information no one really cares about. If the game happened, the time it happened isn't super relevant. People might care about the score or when the game was but beyond that the timestamp was busy. 

![image](https://github.com/jschultz38/degenbot/assets/2738715/b30d2b93-7c03-4e12-a313-c00c4a978474)
 Looked better than 
 
![image](https://github.com/jschultz38/degenbot/assets/2738715/5a016822-0e6c-425f-9b43-b6e7ba0c3400)

It also created a visual difference so it was easier to parse out what was past.